apps/login/src/lib/server/loginname.ts

```diff
@@ -0,0 +1,118 @@
+"use server";
+
+import { headers } from "next/headers";
+import { idpTypeToSlug } from "../idp";
+import {
+  getActiveIdentityProviders,
+  getLoginSettings,
+  listAuthenticationMethodTypes,
+  listUsers,
+  startIdentityProviderFlow,
+} from "../zitadel";
+import { createSessionForUserIdAndUpdateCookie } from "../../utils/session";
+import { redirect } from "next/navigation";
+
+export type SendLoginnameOptions = {
+  loginName: string;
+  authRequestId?: string;
+  organization?: string;
+};
+
+export async function sendLoginname(options: SendLoginnameOptions) {
```

Comment:
the trick is to call it `command`, all auctions are a single parameter function with one argument called `command`